src/app/load-app.js

```diff
@@ -31,7 +52,9 @@ const LoadApp = ({ children }) => {
         )
     }
 
-    return children
+    if (data && (userData || userError)) {
```

Comment:
I set it up this way as it seems we don't want to block the user from using the app if something fails with api/me request, but at the same time it's better to wait for the response to complete...could also refactor this to await query with useDataEngine?